lib/pronto/config_file.rb

```diff
@@ -35,8 +36,10 @@ class ConfigFile
       'format' => DEFAULT_MESSAGE_FORMAT
     }.freeze
 
-    def initialize(path = '.pronto.yml')
-      @path = path
+    def initialize
+      @path = ENV.fetch('PRONTO_CONFIG', DEFAULT_FILE_PATH)
+      return if @path == DEFAULT_FILE_PATH || File.exist?(@path)
```

Comment:
@nbekirov yeah. I think it should still be in there to check if the default file (`.pronto.yml`) exists...

@doomspork I've removed the default option for `--config` and changed the check to always raise an error if it (or the `PRONTO_CONFIG` env variable) is set. If not it continues and sets it to `.pronto.yml` which is handled in the `File.exist?` check @nbekirov is talking about.